src/HotChocolate/Core/test/Types.Scalars.Tests/NegativeIntTypeTests.cs

```diff
@@ -0,0 +1,117 @@
+using System;
+using HotChocolate.Language;
+using Snapshooter.Xunit;
+using Xunit;
+
+namespace HotChocolate.Types.Scalars
+{
+    public class NegativeIntTypeTests : ScalarTypeTestBase
+    {
+        [Fact]
+        public void Schema_WithScalar_IsMatch()
+        {
+            // arrange
+            ISchema schema = BuildSchema<NegativeIntType>();
+
+            // act
+            // assert
+            schema.ToString().MatchSnapshot();
+        }
+
+        [Theory]
+        [InlineData(typeof(EnumValueNode), TestEnum.Foo, false)]
+        [InlineData(typeof(FloatValueNode), 1d, false)]
+        [InlineData(typeof(IntValueNode), -1, true)]
```

Comment:
It would be nice to see the edge cases also tested
```suggestion
        [InlineData(typeof(IntValueNode), -1, true)]
        [InlineData(typeof(IntValueNode), int.MinValue, true)]
        [InlineData(typeof(IntValueNode), 0, false)]
```